scenarios/fuel_test/src/AllEventsHandlers.ts

```diff
@@ -196,7 +198,15 @@ AllEvents.Option2.handler(async ({ event }) => {
 });
 
 const vecLogSchema = S.array(SExtra.bigint);
-AllEvents.Vec.handler(async ({ event }) => {
+AllEvents.VecLog.handler(async ({ event }) => {
   vecLogSchema.assert(event.params)!;
   expectType<AssertSchemaType<typeof event.params, typeof vecLogSchema>>(true);
 });
+
+const bytesLogSchema = S.unknown;
+AllEvents.BytesLog.handler(async ({ event }) => {
+  bytesLogSchema.assert(event.params)!;
+  expectType<AssertSchemaType<typeof event.params, typeof bytesLogSchema>>(
```

Comment:
I think I need a little bit more explanation on this. So it's seen as an unknown type.

Does that mean it's the data exactly raw as is from the chain and it's up to the user to parse this type into whatever they want?

Reply:
> Does that mean it's the data exactly raw as is from the chain and it's up to the user to parse this type into whatever they want?

Yes